app/controllers/application_controller.rb

```diff
@@ -1,4 +1,10 @@
 # frozen_string_literal: true
 
 class ApplicationController < ActionController::Base
+	def require_admin
+		unless current_user && current_user.is_admin
```

Comment:
This can be simplified with the Ruby safe navigation operator

Reply:
changed code